pkg/syncer/k8scloudoperator/k8scloudoperator.go

```diff
@@ -160,6 +161,13 @@ func (k8sCloudOperator *k8sCloudOperator) GetPodVMUUIDAnnotation(ctx context.Con
 			pod, err = k8sCloudOperator.k8sClient.CoreV1().Pods(podNamespace).Get(ctx, podName, metav1.GetOptions{})
 			if err != nil {
 				vmUuidNotFoundError = false
```

Comment:
nit: Should we move `vmUuidNotFoundError = false` to the else block after newly introduced if block? Better readability IMO.

Reply:
Done